### PR TITLE
feat(www): Adds CodeSandbox in Try this starter

### PR DESCRIPTION
**Pull request by garethx**
## Description
This PR adds a CodeSandbox option to 'Try this starter' on starter pages. Like Netlify, you can deploy the starter but with the CodeSandbox option folks will also be able to see and edit a copy of the code to test out changes to the starter too.

### CODE REVIEW

**Comment by pieh** [comment]
I will just check how adding more items to "Try this starter" impacts displayed information on narrow viewports (mostly worried about "mobile/portrait" case).

But general feature to add CodeSandbox has "+1" from me!

**Comment by garethx** [comment]
It looks like that section gets hidden altogether on smaller viewports

**Comment by pieh** [comment]
> It looks like that section gets hidden altogether on smaller viewports

Yeah, seeing same thing. I wonder now about those svg icons that doesn't seem to be displayed anywhere at the moment.

One thing is that it adds "CodeSandbox" link before Netlify, which I'm not sure if something we need to think too much about right now, but worth mentioning:
Screenshot 2020-01-31 at 15 40 26

The CodeSandbox might just make more sense as first option as it's quickly interactive, without having to pull things from fork/clone that Netlify creates on behalf of user.

**Comment by pieh** [comment]
Hmm, the svg imports seems to just come out as `undefined`, but this is not directly related to this Pull Request, so let's ignore it in this PR

**Comment by pieh** [comment]
Ok the `import { CodesandboxIcon } from "../../assets/vendor-logos/codesandbox.svg"` is trying to get named export, it should just grab default one so we should drop `{ }` from those statements - but it results in:
Screenshot 2020-01-31 at 15 53 53

Which (to me) lacks a bit of whitespace between items

**Comment by garethx** [comment]
@pieh Thanks for the feedback. I've got the icons working, and added some whitespace between the options.

**Comment by pieh** [comment]
If you have site running locally - could you paste some screenshots for folks to see? If not, I'll just update mine to use your changes.

**Comment by garethx** [comment]
Screenshot 2020-01-31 at 15 48 20

File: www/src/views/starter/source.js [hunk 1]
```diff
@@ -66,6 +67,17 @@ const Source = ({ startersYaml, repoUrl }) => (
       >
         Try this starter
       </span>
+      <a
+        href={`https://codesandbox.io/s/github/${starter.owner}/${starter.stub}`}
+        sx={{
+          "&&": {
+            borderBottom: 0,
+            marginRight: `5px`
```
Comment (fk) [inline www/src/views/starter/source.js]: Hey @garethx! 🤗Thanks for taking care of this! 🙏 

May I ask you to use a token from our `space` scale (see https://www.gatsbyjs.org/guidelines/design-tokens/, and https://github.com/gatsbyjs/gatsby/tree/master/www#theme-ui-css-authoring-and-dark-mode for a bit of background info), and replace this line with `marginRight: 2`? This also already slightly increases the value from `5px` to `8px`. I'm not 100% sure is enough whitespace to clearly separate CodeSandbox and Netlify and their icons, so could you take a look at `3` (`12px`), and `4`(`16px`), too? 🙏

Comment (garethx) [reply www/src/views/starter/source.js]: What do you think to space[4]? Screenshot 2020-01-31 at 22 15 59

Comment (sidharthachatterjee) [reply www/src/views/starter/source.js]: [4] looks great!

**Pull request merged**